pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -353,7 +353,9 @@ let
     # Filesystem options - in particular, enable extended attributes and
     # ACLs for all filesystems that support them.
     filesystem = {
-      FANOTIFY        = yes;
+      FANOTIFY                    = yes;
+      FANOTIFY_ACCESS_PERMISSIONS = yes;
```

Comment:
Both available on 4.14 trough 6.0, second one appears to default to `no` on upstream linux defaults;

https://www.kernelconfig.io/config_fanotify_access_permissions?q=&kernelversion=4.14&arch=x86

https://www.kernelconfig.io/config_fanotify_access_permissions?q=&kernelversion=6.0&arch=x86

LGTM